bgpd/bgp_attr.h

```diff
@@ -205,6 +205,9 @@ struct attr {
 
 	/* EVPN local router-mac */
 	struct ethaddr rmac;
+
+	/* rmap set table */
+	uint32_t rmap_table_id;
```

Comment:
question: shouldn't we need to update `attrhash_key_make()` and `attrhash_cmp()` if we're adding a new field to the `attr` structure?

Reply:
you are right. I will add this.